backend/controller/adminAuth.go

```diff
@@ -0,0 +1,64 @@
+package controller
+
+import (
+	"api/models"
+	"encoding/json"
+	"github.com/golang-jwt/jwt/v5"
+	"net/http"
+	"time"
+)
+
+var secretKey = []byte("secret") // TODO: put in .env file
```

Comment:
This should go in a .env file

Reply:
Will this break with an empty string? If not, I think it would be worth replacing this with `[]byte(os.Getenv("TOKEN_SECRET"))` in this PR. This will use the environment variable if available, but will return an empty string if not.

Reply:
+1

Reply:
How is the secret accessible on frontend?

Reply:
Also this doesn't work for an empty string i forgot to check earlier

Reply:
> @cameronkinsella did you fix getting the secret from the frontend?

Not sure what you're referring to. Can you elaborate?

Reply:
> Also this doesn't work for an empty string i forgot to check earlier

Okay, if it doesn't work with "" instead of "secret" then I think it's fine to leave it as "secret" for now. 

We can add a helper function in a later PR to manage env variables with default values a bit better. We can use it for what I added in #27 to clean that up as well.

Reply:
> > @cameronkinsella did you fix getting the secret from the frontend?
> 
> Not sure what you're referring to. Can you elaborate?

i thought you mentioned there was an issue getting environment variables into the frontend at one point.

Reply:
Oh, yeah that's right. The fix for this was added in #27. I added a guide for how to add new ones in the frontend's README.

Reply:
ok cool, not changing anything for this PR though

Reply:
yeah, that's fine.